src/channelape/controller/ChannelApeActionsController.ts

```diff
@@ -10,36 +10,36 @@ const PARSE_INT_RADIX = 10;
 
 export default abstract class ChannelApeActionsController {
   protected logger: Logger;
-  private updateHealthCheckInterval?: NodeJS.Timer;
 
   constructor(loggerName: string, protected channelApeClient: ChannelApeClient) {
     this.logger = new Logger(loggerName, Secrets.env.LOG_LEVEL);
   }
 
-  protected abstract processAction(businessId: string, actionId: string): void;
+  protected abstract processAction(businessId: string, actionId: string): Promise<any>;
 
-  public handle(req: Request, res: Response): Promise<void | Response> {
+  public async handle(req: Request, res: Response): Promise<void | Response> {
```

Comment:
Can this return `Promise<void>`? What reason is there to return a response here? looks like we are already sending responses within this function.